pkg/controller/registry/reconciler/reconciler_test.go

```diff
@@ -74,3 +74,20 @@ func TestPullPolicy(t *testing.T) {
 		}
 	}
 }
+
+func TestPodRunAsUser(t *testing.T) {
+	source := &v1alpha1.CatalogSource{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "test",
+			Namespace: "test-ns",
+		},
+	}
+
+	p := Pod(source, "hello", "busybox", "", map[string]string{}, int32(0), int32(0))
```

Comment:
It looks like you're missing arguments according to that failed unit test:

```
INFO[2021-05-03T18:49:03Z] pkg/controller/registry/reconciler/reconciler_test.go:86:10: not enough arguments in call to Pod 
INFO[2021-05-03T18:49:03Z] 	have (*"github.com/operator-framework/api/pkg/operators/v1alpha1".CatalogSource, string, string, string, map[string]string, int32, int32) 
INFO[2021-05-03T18:49:03Z] 	want (*"github.com/operator-framework/api/pkg/operators/v1alpha1".CatalogSource, string, string, string, map[string]string, map[string]string, int32, int32) 
```

Reply:
Should be updated in https://github.com/operator-framework/operator-lifecycle-manager/pull/2023/commits/f8df548373274250ae72e248f8639707998adccb